services/jupyterhub.qmd

```diff
@@ -6,23 +6,23 @@ The US GHG Center promotes the use of JupyterHub environments for interactive da
+If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.GitHub.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.
 
 
 ## To Get US GHG Center JupyterHub access:
 
- The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our Hub Access Request form.following these steps:
+ The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7)
 
-* Make sure you have a [Github Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [Github username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).
-* Fill out the request form and provide needed information.
-* Watch your email for notification of authorization and the invite to join the [US GHG Center Hub Access Github Team](https://github.com/orgs/US-GHG-Center/teams/ghgc-hub-access).
+* Make sure you have a [GitHub Account](https://docs.GitHub.com/en/get-started/signing-up-for-GitHub/signing-up-for-a-new-GitHub-account). Take note of your [GitHub username](https://docs.GitHub.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-GitHub/managing-email-preferences/remembering-your-GitHub-username-or-email).
+* Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.
+* Watch your email for notification of authorization and the invite to join the [US GHG Center Hub Access GitHub Team](https://GitHub.com/orgs/US-GHG-Center/teams/ghgc-hub-access).
 
-* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your Github credentials.
+* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your GitHub credentials.
 
 
 ## To access User Notebooks
 
 This site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) showing how to load and analyze Earth data in the interactive cloud computing environment.
 
-Further instructions are included in each notebook.  
+Further instructions are included in each notebook.
```

Comment:
add a line break after "notebook."

this all would have been super easy if you had just accepted my suggestions directly